src/Components/LoginPage.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import CustomDiv from '../CustomComponents/CustomDiv';
+import CustomInput from '../CustomComponents/CustomInput';
+import CustomButton from '../CustomComponents/CustomButton';
+import CustomForm from "../CustomComponents/CustomForm";
+import { userStore } from '../Stores/UserStore';
+import { useNavigate } from "react-router-dom";
+
+const LoginPage: React.FC = () => {
+  const navigate = useNavigate();
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
in the store please like  we have agrren in previousproject

Reply:
Done